runners/pom.xml

```diff
@@ -55,6 +55,13 @@
             <plugin>
               <groupId>org.apache.maven.plugins</groupId>
               <artifactId>maven-surefire-plugin</artifactId>
+              <dependencies>
+                <dependency>
+                  <groupId>org.apache.beam</groupId>
+                  <artifactId>beam-sdks-java-core</artifactId>
+                  <type>test-jar</type>
```

Comment:
Shouldn't we depend on the shaded version of the test jar and not the vanilla one since beam-sdks-java-core does shading?